python/pyspark/ml/classification.py

```diff
@@ -896,23 +896,27 @@ class GBTClassifier(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol
                      "Supported options: " + ", ".join(GBTParams.supportedLossTypes),
                      typeConverter=TypeConverters.toString)
 
+    subsamplingRate = Param(Params._dummy(), "subsamplingRate", "Fraction of the training " +
```

Comment:
I'm sorry, I hadn't realized that this was already inherited from TreeEnsembleParams.  Your original PR was correct.